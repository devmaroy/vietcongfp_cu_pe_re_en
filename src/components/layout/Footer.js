import React from 'react';

const Footer = () => {
    const iconsCopyright = (
        <div>
            Icons made by  
            <a href="https://www.freepik.com/" title="Freepik"> Freepik </a> 
             from <a href="https://www.flaticon.com/" title="Flaticon"> www.flaticon.com </a> 
             is licensed by 
            <a href="http://creativecommons.org/licenses/by/3.0/" title="Creative Commons BY 3.0" target="_blank"> CC 3.0 BY </a>
        </div>
    );

    return (
        <div className="content-container">
            <div className="footer">
                <p>Copyright &copy; { new Date().getFullYear() }</p>
                <p>All rights reserved.</p>
                <p>Made with <i className="heart-symbol"></i> by <a href="https://marekmatejovic.com/">devmaroy</a></p> 
                { iconsCopyright }
            </div>
        </div>
    );
};

export default Footer;