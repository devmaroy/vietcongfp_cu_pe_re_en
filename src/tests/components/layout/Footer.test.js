import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/layout/Footer';

test( 'should correctly render Footer', () => {
    const wrapper = shallow( <Footer /> );

    expect( wrapper ).toMatchSnapshot();
});