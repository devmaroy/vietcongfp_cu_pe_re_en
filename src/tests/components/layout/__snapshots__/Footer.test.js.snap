// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Footer 1`] = `
<div
  className="content-container"
>
  <div
    className="footer"
  >
    <p>
      Copyright Â© 
      2019
    </p>
    <p>
      All rights reserved.
    </p>
    <p>
      Made with 
      <i
        className="heart-symbol"
      />
       by 
      <a
        href="https://mwardew.com/"
      >
        MWardew
      </a>
    </p>
    <div>
      Icons made by
      <a
        href="https://www.freepik.com/"
        title="Freepik"
      >
         Freepik 
      </a>
      from 
      <a
        href="https://www.flaticon.com/"
        title="Flaticon"
      >
         www.flaticon.com 
      </a>
      is licensed by
      <a
        href="http://creativecommons.org/licenses/by/3.0/"
        target="_blank"
        title="Creative Commons BY 3.0"
      >
         CC 3.0 BY 
      </a>
    </div>
  </div>
</div>
`;
